app/Console/Commands/GetWpOrders.php

```diff
@@ -0,0 +1,158 @@
+<?php
+
+namespace App\Console\Commands;
+
+use Illuminate\Console\Command;
+use App\Models\Dashboard;
+use App\Models\Orders;
+use App\Models\OrderLineItems;
+use Automattic\WooCommerce\Client;
+
+class UpdateOrders extends Command
+{
+    /**
+     * The name and signature of the console command.
+     * time ['all', 'today']
+     * @var string
+     */
+    protected $signature = 'orders:update {time_report?}';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Update Orders';
+
+    /**
+     * Execute the console command.
+     *
+     * @return int
+     */
+    public function handle()
+    {
+        $this->info("Cron Job Update Wordpress Orders running at " . now());
+        $stores = Dashboard::getWooStoresList();
+
+        foreach ($stores as $store) {
+            $wooConfig = Dashboard::getWooStoresList($store);
+
+            $domain = $wooConfig['domain'];
+            $consumerKey = $wooConfig['consumerKey'];
+            $consumerSecret = $wooConfig['consumerSecret'];
+            $apiVersion = $wooConfig['apiVersion'];
+            $dateTimeZone = $wooConfig['dateTimeZone'];
+
+            $dateTime = new \DateTime("now", $dateTimeZone);
+            $dateTime->modify('-1 day');
+            $updatedAtMin = $dateTime->format('Y-m-d');
+
+            $timeReport = $this->argument('time_report') ?? '';
+            if ($timeReport == 'all') {
+                $updatedAtMin = '1900-01-01';
+            }
+
```

Comment:
Xóa code nếu không dùng đến:

$dateTime = new \DateTime("now", $dateTimeZone);
$dateTime->modify('-1 day');
$updatedAtMin = $dateTime->format('Y-m-d');

$timeReport = $this->argument('time_report') ?? '';
if ($timeReport == 'all') {
    $updatedAtMin = '1900-01-01';
}